README.md

```diff
@@ -23,11 +23,11 @@ The ably-python client has one dependency,
 
 ### From a git url
 
-    pip install git@bitbucket.org:jjwchoy/ably-python.git
+    pip install -e git+https://github.com/ably/ably-python#egg=AblyPython
```

Comment:
It's the syntax if you want to install a package from GitHub with pip https://pip.pypa.io/en/latest/reference/pip_install.html#git and you will find more explanations about the suffix with #egg here https://pip.pypa.io/en/latest/reference/pip_install.html#vcs-support